specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/disk.json

```diff
@@ -0,0 +1,1448 @@
+      "properties": {
+        "osType": {
+          "type": "string",
+          "description": "the Operating System type.",
+          "enum": [
+            "Windows",
+            "Linux"
+          ],
+          "x-ms-enum": {
+            "name": "OperatingSystemTypes",
+            "modelAsString": false
+          }
+        },
+        "diskSizeGB": {
+          "type": "integer",
+          "format": "int32",
+          "description": "If creationData.createOption is Empty, this field is mandatory and it indicates the size of the VHD to create. If this field is present for updates or creation with other options, it indicates a resize. Resizes are only allowed if the disk is not attached to a running VM, and can only increase the disk's size."
+        },
+        "diskSizeBytes": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "The size of the disk in bytes. This field is read only."
+        },
+        "encryptionSettingsCollection": {
+          "$ref": "#/definitions/EncryptionSettingsCollection",
+          "description": "Encryption settings collection used be Azure Disk Encryption, can contain multiple encryption settings per disk or snapshot."
+        }
+      },
+      "description": "Snapshot resource update properties."
+    },
+    "CreationData": {
+      "properties": {
+        "createOption": {
+          "type": "string",
+          "enum": [
+            "Empty",
+            "Attach",
+            "FromImage",
+            "Import",
+            "Copy",
+            "Restore",
+            "Upload"
+          ],
+          "x-ms-enum": {
+            "name": "DiskCreateOption",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Empty",
+                "description": "Create a new disk with a given size."
+              },
+              {
+                "value": "Attach",
+                "description": "Disk will be attached to a VM."
+              },
+              {
+                "value": "FromImage",
+                "description": "Create a new disk by importing from a PIR image or a customer's user image."
+              },
+              {
+                "value": "Import",
+                "description": "Create a new disk by importing from a customer's blob. The blob must be formatted as a VHD."
+              },
+              {
+                "value": "Copy",
+                "description": "Create a new disk by copying from a customer's managed disk or snapshot."
+              },
+              {
+                "value": "Restore",
+                "description": "Create a new disk by restoring from a restore point."
+              },
+              {
+                "value": "Upload",
+                "description": "Create a new disk by customer upload using a write SAS to drectly write the contents of their disk to the blob. The contents must be formatted as a VHD."
```

Comment:
```suggestion
                "description": "Create a new disk by customer upload using a write SAS to directly write the contents of their disk to the blob. The contents must be formatted as a VHD."
```

Reply:
Yeah I saw the spell checker validation failed and fixed this